src/main/java/com/library/model/entity/Author.java

```diff
@@ -0,0 +1,51 @@
+package com.library.model.entity;
+
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Transient;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Author extends CustomEntity {
+
+    @Column
+    private String name;
+
+    @Column
+    private String surname;
+
+    @Transient
+    private String fullName;
+
+    @Column(name = "date_of_birth")
+    private LocalDate birthDay;
+
+    @Column(name = "date_of_death")
+    private LocalDate ripDay;
+
+    @ManyToMany(mappedBy = "authors", fetch = FetchType.LAZY)
+    private List<Book> books;
+
+    @PostLoad
+    private void setFullName(){
+        this.fullName = name.concat(" ").concat(surname);
+    }
```

Comment:
Never seen this in real life)
And even can't imagine why this annotation is needed in entity.

Reply:
I wanted to use annotation @Formula, but it doesn't work for me and I founded this way

Reply:
Will discuss on call today